### PR TITLE
Home card updates

### PR DESCRIPTION
**Pull request by ice-atlas**

### CODE REVIEW

File: src/screens/Modals/ProfilePrivacyEdit/step2/index.tsx [hunk 1]
```diff
@@ -35,10 +35,10 @@ export const ProfilePrivacyEditStep2 = () => {
         />
         <View style={styles.roleContainer}>
           <CurrentRoleCard
-            imageSource={Images.roles.pioneer}
-            imageSourceHidden={Images.roles.pioneerInactive}
-            title={t('role.pioneer.title')}
-            description={t('role.pioneer.description')}
+            imageSource={Images.roles.snowman}
```
Comment (ice-ares) [inline src/screens/Modals/ProfilePrivacyEdit/step2/index.tsx:38]: No, this is generic, use the type, dont hardcode anything

File: src/screens/ProfileFlow/Profile/components/Role/index.tsx [hunk 2]
```diff
@@ -27,9 +27,9 @@ export const Role = memo(({user}: Props) => {
         <CurrentRoleSkeleton />
       ) : (
         <CurrentRoleCard
-          imageSource={Images.roles.pioneer}
-          imageSourceHidden={Images.roles.pioneerInactive}
-          title={'Pioneer'}
+          imageSource={Images.roles.snowman}
+          imageSourceHidden={Images.roles.snowmanInactive}
+          title={'Snowman'}
```
Comment (ice-ares) [inline src/screens/ProfileFlow/Profile/components/Role/index.tsx:32]: This is also generic

File: src/screens/ProfileFlow/Roles/index.tsx [hunk 3]
```diff
@@ -42,10 +42,10 @@ export const Roles = () => {
         showsVerticalScrollIndicator={false}
         contentContainerStyle={[styles.scrollContent, bottomOffset.current]}>
         <Role
-          title="Pioneer"
+          title="Snowman"
```
Comment (ice-ares) [inline src/screens/ProfileFlow/Roles/index.tsx:45]: Should be fetched from list

File: src/store/modules/Achievements/reducer/index.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+import {RoleType} from '@api/achievements/types';
+import {AccountActions} from '@store/modules/Account/actions';
+import {AchievementsActions} from '@store/modules/Achievements/actions';
+import produce from 'immer';
+
+interface State {
```
Comment (ice-ares) [inline src/store/modules/Achievements/reducer/index.ts:8]: Where's the tasks reducer placed?

**Review by ice-ares — COMMENTED** [review]
You either rename the Achievements module to LevelsAndRoles or move all the tasks related stuff under Achievements as well

File: src/screens/Modals/ProfilePrivacyEdit/step2/index.tsx [hunk 5]
```diff
@@ -35,10 +39,14 @@ export const ProfilePrivacyEditStep2 = () => {
         />
         <View style={styles.roleContainer}>
           <CurrentRoleCard
-            imageSource={Images.roles.pioneer}
-            imageSourceHidden={Images.roles.pioneerInactive}
-            title={t('role.pioneer.title')}
-            description={t('role.pioneer.description')}
+            imageSource={Images.roles[roleType]}
+            imageSourceHidden={
+              roleType !== 'ambassador'
```
Comment (ice-ares) [inline src/screens/Modals/ProfilePrivacyEdit/step2/index.tsx:44]: Different from the enabled one

Comment (ice-atlas) [reply src/screens/Modals/ProfilePrivacyEdit/step2/index.tsx:44]: Why we need selector for this? In this particular case we just do not have inactive image for ambassador.

File: src/api/achievements/getLevelsAndRoles.ts [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+interface LevelsAndRoles {
+  level: number;
+  roles: Role[];
+}
+
+export function getLevelsAndRoles({userId}: Params) {
+  return get<LevelsAndRoles>(
+    `/achievement-summaries/levels-and-roles/users/${userId}`,
```
Comment (ice-hector) [inline src/api/achievements/getLevelsAndRoles.ts:18]: @ice-ares Typo in api naming? According to type above name should be level-and-roles maybe?

File: src/api/achievements/index.ts [hunk 7]
```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+import {getLevelsAndRoles} from './getLevelsAndRoles';
+
+export const achievements = Object.freeze({
+  getLevelsAndRoles,
```
Comment (ice-hector) [inline src/api/achievements/index.ts:6]: getLevelAndRoles ?

File: src/api/achievements/getLevelsAndRoles.ts [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+import {get} from '@api/client';
+
+import {Role} from './types';
+
+interface Params {
+  userId: string;
+}
+
+interface LevelsAndRoles {
```
[the remark this answers is not in the document]
Comment (ice-atlas) [reply src/api/achievements/getLevelsAndRoles.ts:11]: exactly

File: src/screens/HomeFlow/Home/components/Overview/components/LevelCard/index.tsx [hunk 9]
```diff
@@ -38,9 +43,9 @@ export const LevelCard = ({isCollapsed}: Props) => {
   return (
     <CardBase
       backgroundImageSource={Images.backgrounds.levelCardBg}
-      headerTitle={t('home.pioneer.title')}
+      headerTitle={t(`roles.${roleType}.title`)}
       headerTitleIcon={<PioneerIcon fill={COLORS.white} />}
-      headerValue={t('global.level') + ' 1'}
+      headerValue={`${t('global.level')} ${userLevel}`}
```
Comment (ice-hector) [inline src/screens/HomeFlow/Home/components/Overview/components/LevelCard/index.tsx:48]: Will that work with RTL?

File: src/screens/Modals/ProfilePrivacyEdit/step2/index.tsx (hunk 5 above)
Comment (ice-atlas) [reply src/screens/Modals/ProfilePrivacyEdit/step2/index.tsx]: Done

File: src/screens/HomeFlow/Home/components/Overview/components/LevelCard/index.tsx (hunk 9 above)
Comment (ice-atlas) [reply src/screens/HomeFlow/Home/components/Overview/components/LevelCard/index.tsx:48]: will previous solution work? This is out of scope of this ticket. Just adjusted to use real data. RTL will be managed separately

**Review by ice-ares — APPROVED** [review]

File: src/translations/locales/en.json [hunk 10]
```diff
@@ -641,9 +640,13 @@
   "country_select": {
     "title": "Select country"
   },
-  "role": {
-    "pioneer": {
-      "title": "Pioneer",
+  "roles": {
+    "ambassador": {
+      "title": "AMBASSADOR",
+      "description": "AMBASSADOR_description"
```
[the remark this answers is not in the document]
Comment (ice-ares) [reply src/translations/locales/en.json:646]: @ice-atlas ask the business for a value there

File: src/api/achievements/index.ts (hunk 7 above)
Comment (ice-atlas) [reply src/api/achievements/index.ts:6]: no, it is based on api naming

File: src/translations/locales/en.json (hunk 10 above)
[the remark this answers is not in the document]
Comment (ice-atlas) [reply src/translations/locales/en.json]: fixed

**Review by ice-ares — APPROVED** [review]

**Review by ice-hector — APPROVED** [review]

**Pull request merged**